tests/e2e/operator/operator_test.go

```diff
@@ -123,14 +123,10 @@ var _ = Describe("Operator", Ordered, func() {
 		It("updates the CRDs status to Established", func() {
 			for _, crd := range crds {
 				Eventually(func(g Gomega) {
-					Expect(k8sClient.Get(context.TODO(), client.ObjectKey{Namespace: namespace, Name: crd}, customResourceDefintion)).
+					Expect(k8sClient.Get(context.TODO(), key(namespace, crd), customResourceDefintion)).
```

Comment:
Since CRDs are cluster-scoped, this should actually call `key("", crd)`. But maybe we should introduce a new key function for cluster-scoped resources.